.eslintrc.js

```diff
@@ -4,6 +4,7 @@ module.exports = {
   rules: {
     curly: 'error',
     '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-inferrable-types': 'off',
```

Comment:
The `typedef` rule was upset that the `shouldListen` prop did not have a definition, and when I added one, this rule was upset about that. I feel like it's OK if we define types that are inferable? It doesn't seem like it hurts anything IMO, it seems like it's extra clarity.